src/dotnet-aspnet-codegenerator/DotNetBuildCommandHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Cli.Utils;
+using NuGet.Frameworks;
+
+namespace Microsoft.Extensions.CodeGeneration
+{
+    internal class DotNetBuildCommandHelper
+    {
+        internal static int Build(
```

Comment:
Yes. As long as we are trying to share implementation, we may as well move this too.